fe/fe-core/src/main/java/com/starrocks/lake/delete/LakeDeleteJob.java

```diff
@@ -96,7 +96,7 @@ public void run(DeleteStmt stmt, Database db, Table table, List<Partition> parti
             }
             throw new DdlException(t.getMessage(), t);
         } finally {
-            locker.unLockDatabase(db, LockType.READ);
+            locker.unLockTablesWithIntensiveDbLock(db, Lists.newArrayList(table.getId()), LockType.READ);
```

Comment:
is it possible to provide an interface for single table lock, it should be pretty common locking a single table. `Lists.newArrayList` looks so redundant just for adapting the interface.

Reply:
It's possible in the future, but not this PR. table-level-lock is still in the initial trial stage, and I don’t want to introduce too many interfaces.